### PR TITLE
Add accessibility to filtered arguments

### PR DESCRIPTION
**Pull request by groony**

### CODE REVIEW

**Comment by groony** [comment]
can anybody merge my pull request or ask questions?

**Comment by piton4eg** [comment]
@groony sorry for delay. Could you describe how are you going to use it? Or even better add update README with it?

**Comment by groony** [comment]
@piton4eg Ok. README updated now.

**Comment by kimrgrey** [comment]
Looks great! Let's add small "this thing is working" test for it and ready to be merged.

**Comment by groony** [comment]
add small tests for it

File: spec/lib/performify/initialize_spec.rb [hunk 1]
```diff
@@ -25,12 +25,20 @@
       expect(subject.foo).to eq(args[:foo])
     end
 
+    it 'no filtered inputs' do
+      expect(subject.inputs).to be_nil
```
Comment (piton4eg) [inline spec/lib/performify/initialize_spec.rb]: Hm, I think that when we don't have validation `schema` for arguments, then maybe `inputs` should have all arguments? 
@kimrgrey @groony what do you think?

Comment (groony) [reply spec/lib/performify/initialize_spec.rb]: @piton4eg I think it will be duplication of attribute `args `

Comment (kimrgrey) [reply spec/lib/performify/initialize_spec.rb]: @piton4eg LGTM

**Comment by kimrgrey** [comment]
Released and pushed gem: https://rubygems.org/gems/performify/versions/0.6.0. Thanks @groony !

**Pull request merged**